### ISSUE TITLE
Can't run the End-to-End Pipeline Example on Azure

### ISSUE THREAD

**Issue by lalithvaka** [kubeflow/examples#670]
Have Kubeflow 0.62 v on AKS cluster. Started following the steps listed under https://www.kubeflow.org/docs/azure/azureendtoend/

Preprocess step completed and the training failed with exit code 1.  Log captured the following. Not sure if the example code has some bug that needs rectification.

loading dataset from /mnt/azure/train.txt
dataset size: 559
suffling data...
splitting data...
2019-10-24 20:38:06.449997: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Could not dlopen library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIRARY_PATH: /usr/local/cuda/extras/CUPTI/lib64:
2019-10-24 20:38:06.450040: E tensorflow/stream_executor/cuda/cuda_driver.cc:318] failed call to cuInit: UNKNOWN ERROR (303)
2019-10-24 20:38:06.450077: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:153] kernel driver does not appear to be running on this host (tacos-vs-burritos-6qxrg-3296651659): /proc/driver/nvidia/version does not exist
2019-10-24 20:38:06.467692: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2294680000 Hz
2019-10-24 20:38:06.467957: I tensorflow/compiler/xla/service/service.cc:162] XLA service 0x545d0c0 executing computations on platform Host. Devices:
2019-10-24 20:38:06.467982: I tensorflow/compiler/xla/service/service.cc:169]   StreamExecutor device (0): <undefined>, <undefined>
Traceback (most recent call last):
  File "/scripts/train.py", line 188, in <module>
    model_signature = run(**args)
  File "/scripts/train.py", line 85, in run
    Dataset.from_tensor_slices(list(train_labels)), img_size))
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/data/ops/dataset_ops.py", line 591, in zip
    return ZipDataset(datasets)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/data/ops/dataset_ops.py", line 2510, in __init__
    raise TypeError(message)
TypeError: The argument to `Dataset.zip()` must be a nested structure of `Dataset` objects.

![image](https://user-images.githubusercontent.com/8292387/67525159-00fa9f80-f667-11e9-8cab-540f405eb0ef.png)

**Comment by umialpha** [comment]
I got the same error, It was a syntax error. But I am not familiar with tf.

**Comment by umialpha** [comment]
Hi, @lalithvaka I have fixed this bug. change `train_ds = Dataset.zip((Dataset.from_tensor_slices(list(train_data)),
                          Dataset.from_tensor_slices(list(train_labels)),  img_zie))` to `train_ds = Dataset.zip((Dataset.from_tensor_slices(list(train_data)),
                          Dataset.from_tensor_slices(list(train_labels)),  Dataset.from_tensor_slices([img_size] * len(train_data))))`.
But I have to tell you, there are other several more bugs in this tutorial ;(

@jlewi Hi, I have already fixed this error on-premise. BTW, there are a lot bugs in azurepipeline errors. I don't know whether it is because of API incompatibility or lack of autiousness. Anyway, I fixed them all and run the whole pipeline successfully. Could I commit my fix?
